samples/analyzeIamPolicyLongrunningGcs.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// sample-metadata:
+//   title: Analyze Iam Policy Longrunning and write results to GCS
+//   description: Analyzes accessible IAM policies that match a request.
+//   usage: node analyzeIamPolicyLongrunningGcs
+//   <gs://my-bucket/my-analysis.json>
+
+async function main(gcsUri) {
+  // [START asset_quickstart_analyze_iam_policy_longrunning_gcs]
+  const util = require('util');
+  const {AssetServiceClient} = require('@google-cloud/asset');
+
+  const client = new AssetServiceClient();
+  const projectId = await client.getProjectId();
+
+  async function analyzeIamPolicyLongrunningGcs() {
+    // TODO(developer): choose the gcs path uri
+    // const gcsUri = 'Gcs path uri, e.g.: gs://<my_bucket>/<my_analysis_file>'
+
+    const request = {
+      analysisQuery: {
+        scope: `projects/${projectId}`,
+        resourceSelector: {
+          fullResourceName: `//cloudresourcemanager.googleapis.com/projects/${projectId}`,
+        },
+        options: {
+          expandGroups: true,
+          outputGroupEdges: true,
+        },
+      },
+      outputConfig: {
+        gcsDestination: {
+          uri: gcsUri,
+        },
+      },
+    };
+
+    // Handle the operation using the promise pattern.
+    const [operation] = await client.analyzeIamPolicyLongrunning(request);
+
+    // Operation#promise starts polling for the completion of the operation.
+    const [result] = await operation.promise();
+
+    // Do things with with the response.
+    console.log(util.inspect(result, {depth: null}));
+  }
+  // [END asset_quickstart_analyze_iam_policy_longrunning_gcs]
+  analyzeIamPolicyLongrunningGcs().catch(err => {
```

Comment:
this will suppress the exit code when the script fails, it might be better to use an approach like this:

https://github.com/googleapis/nodejs-data-qna/blob/35ad521b43b896152c3652b3b265b66d6c66b9cd/samples/quickstart.js#L29

Where we catch an `unhandledRejection` and just call `analyzeIamPolicyLongrunningGcs()` without the catch.